scripts/core/ui/components/SelectUser.tsx

```diff
@@ -9,29 +9,54 @@ import {UserAvatar} from 'apps/users/components/UserAvatar';
 interface IState {
     fetchedUsers?: Array<IUser>;
     selectedUser?: IUser;
+    loading: boolean;
 }
 
 export class SelectUser extends React.Component<IPropsSelectUser, IState> {
     constructor(props: IPropsSelectUser) {
         super(props);
 
-        this.state = {};
+        this.state = {
+            loading: false,
+        };
+
+        this.queryUsers = this.queryUsers.bind(this);
     }
 
-    componentDidMount() {
-        dataApi.query<IUser>('users', 1, {field: 'display_name', direction: 'ascending'}, {})
+    queryUsers(_searchString: string = '') {
+        const searchString = _searchString.trim();
+
+        this.setState({loading: true, fetchedUsers: null});
+
+        dataApi.query<IUser>(
+            'users',
+            1,
+            {field: 'display_name', direction: 'ascending'},
+            (
+                searchString.length > 0
+                    ? {
+                        display_name: {
```

Comment:
I think it should be possible to search by username as well.

Reply:
:+1:  Good catch, I forgot about that feature.